mojaloop/values.yaml

```diff
@@ -4511,197 +6189,448 @@ account-lookup-service:
-      # resources, such as Minikube. If you do want to specify resources, uncomment the following
-      # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-      # limits:
-      #  cpu: 100m
-      #  memory: 128Mi
-      # requests:
 
-quoting-service:
-  # Declare variables to be passed into your templates.
+centraleventprocessor:
+  enabled: false
+
+  image:
+    registry: "docker.io"
+    repository: mojaloop/central-event-processor
+    tag: v12.0.0
+    ## Specify a imagePullPolicy
+    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+    ##
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ## e.g:
+    ## pullSecrets:
+    ##   - myRegistryKeySecretName
+    ##
+    pullSecrets: []
+
   replicaCount: 1
+  command: '["node", "src/index.js"]'
+
+  ## Enable diagnostic mode in the deployment
+  ##
+  diagnosticMode:
+    ## @param diagnosticMode.enabled Enable diagnostic mode (all probes will be disabled and the command will be overridden)
+    ##
+    enabled: false
+    ## @param diagnosticMode.command Command to override all containers in the deployment
+    ##
+    command: 
+      - node
+      - src/index.js
+    ## @param diagnosticMode.args Args to override all containers in the deployment
+    ##
+    args:
+      - --inspect=0.0.0.0:{{ .Values.diagnosticMode.debug.port }}
+    
+    ## @param diagnosticMode.debug config to override all debug information
+    ##
+    debug:
+      internalPort: 9229
+      port: 9229
+
+  readinessProbe:
+    enabled: true
+    httpGet:
+      path: /health
+    initialDelaySeconds: 30
+    periodSeconds: 15
+
+  livenessProbe:
+    enabled: true
+    httpGet:
+      path: /health
+    initialDelaySeconds: 30
+    periodSeconds: 15
+
+  ## Pod scheduling preferences.
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  affinity: {}
+
+  ## Node labels for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+  nodeSelector: {}
+
+  ## Set toleration for scheduler
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  tolerations: []
+
+  config:
+    ## Kafka Configuration
+    # this can be set if the dependency chart for kafka is disabled. If 'kafka_host' is commented out, then the name of the dependency chart will be used.
+    kafka_host: *KAFKA_HOST
+    kafka_port: *KAFKA_PORT
+    mongo_host: *CEP_MONGO_HOST
+    mongo_port: *CEP_MONGO_PORT
+    mongo_user: *CEP_MONGO_USER
+    ## Secret-Management
+    ### Set this if you are using a clear password configured in the config section
+    mongo_password: *CEP_MONGO_PASSWORD
```

Comment:
```suggestion
    mongo_password: ''
```

Reply:
@elnyry-sam-k this is arelady set on the top-level config.